extensions/mssql/src/features.ts

```diff
@@ -27,6 +29,41 @@ export class TelemetryFeature implements StaticFeature {
 	}
 }
 
+export class AccountFeature implements StaticFeature {
+
+	constructor(private _client: SqlOpsDataClient) { }
+
+	fillClientCapabilities(capabilities: ClientCapabilities): void {
+		Utils.ensure(capabilities, 'telemetry')!.telemetry = true;
```

Comment:
Not necessary

Reply:
Oh yes, Ctrl+P Ctrl+V and forgot to remove. Thanks!

Reply:
Left the function body empty except for a comment noting that implementation is not necessary.